backend/cn/main.ml

```diff
@@ -421,49 +438,34 @@ module Executable_spec_flags = struct
 let output_decorated_dir =
   let doc = "output a version of the translation unit decorated with C runtime
   translations of the CN annotations to the provided directory" in
-  Arg.(value & opt (some string) None & info ["output_decorated_dir"] ~docv:"FILE" ~doc)
+  Arg.(value & opt (some string) None & info ["output-decorated-dir"] ~docv:"FILE" ~doc)
 
 let output_decorated =
   let doc = "output a version of the translation unit decorated with C runtime
   translations of the CN annotations." in
-  Arg.(value & opt (some string) None & info ["output_decorated"] ~docv:"FILE" ~doc)
+  Arg.(value & opt (some string) None & info ["output-decorated"] ~docv:"FILE" ~doc)
 
 let with_ownership_checking =
   let doc = "Enable ownership checking within CN runtime testing" in
-  Arg.(value & flag & info ["with_ownership_checking"] ~doc)
+  Arg.(value & flag & info ["with-ownership-checking"] ~doc)
 
 let copy_source_dir =
   let doc = "Copy non-CN annotated files into output_decorated_dir for CN runtime testing" in
-  Arg.(value & flag & info ["copy_source_dir"] ~doc)
+  Arg.(value & flag & info ["copy-source-dir"] ~doc)
```

Comment:
@rbanerjee20 is this ok or should it be reverted so that scripts/workflows can be updated?

Reply:
sure, can update them after Zain has merged since they are not part of the Cerberus CI yet?